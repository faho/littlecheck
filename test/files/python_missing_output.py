# RUN: /usr/bin/python %s

import sys

print("First")
# CHECK: First

# CHECK: NotPrinted

print("Second")
# CHECK: Second
print("Third")
# CHECK: Third

print("1")
# CHECK: {{\d}}
print("2")
# CHECK: {{\d}}
print("3")
# CHECK: {{\d}}
print("4")
# CHECK: {{\d}}
print("5")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}

print("Fourth")
