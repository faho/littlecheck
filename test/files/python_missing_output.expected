Failure in python_missing_output.py:

  The CHECK on line 8 wants:
    NotPrinted

  which failed to match line stdout:2:
    Second

  Context:
    First
     <= nothing to match 'NotPrinted'
    Second
    Third
    1 (nothing to match)
    2
    3
    4
    [...] from line 34 stdout:14
    6
    6
    6
    Fourth <= does not match '{{\\d}}'

  when running command:
    /usr/bin/python python_missing_output.py
